src/utils/utils.ts

```diff
@@ -25,7 +26,15 @@ const mentionRegEx = /((.)\[([^[]*)]\(([^(^)]*)\))/gi;
 
 const defaultMentionTextStyle: StyleProp<TextStyle> = {fontWeight: 'bold', color: 'blue'};
 
-const defaultPlainStringGenerator = ({trigger}: MentionPartType, {name}: MentionData) => `${trigger}${name}`;
+const defaultPlainStringGenerator = ({ trigger }: { trigger?: string | undefined }, { name }: { name?: string }) : string => {
+  if (!trigger || !name) {
+    return '';
```

Comment:
Previously we were [returning `undefined`](https://github.com/venn-city/mobile-app-v2/blob/master/patches/react-native-controlled-mentions%2B2.2.5.patch#L20) , but that doesn't seem like a big deal to return empty string, and that will help to fix typing